app/components/teachers/details/induction_summary_component.rb

```diff
@@ -17,14 +17,18 @@ def extensions_action_text
       (induction_extensions.extended?) ? 'View' : 'Add'
     end
 
-    def render_extension_links?
-      is_admin == false
-    end
-
     def render_add_induction_button?
       is_admin == true
     end
 
+    def extensions_path
+      if is_admin
+        helpers.admin_teacher_extensions_path(teacher)
+      else
+        helpers.ab_teacher_extensions_path(teacher)
+      end
+    end
```

Comment:
I think we should split these out into separate components now to avoid this kind of complexity within.